astro-docs/src/content/docs/technologies/dotnet/introduction.mdoc

```diff
@@ -90,6 +90,12 @@ nx add @nx/dotnet
 The `@nx/dotnet` plugin only became available in Nx 22.0.0. Your Nx Workspace will need to be on this version or higher to use it.
 {% /aside %}
 
+The `@nx/dotnet` plugin doesn't come with a generator. Rather use the `dotnet` CLI to create your app. For example:
```

Comment:
This reads like a downside. We could probably reword it to be a bit more neutral, and link to msft's docs.

```suggestion
New projects can be created with the `dotnet new` CLI command, or any other method you may be familiar with. Full project creation docs can be found here: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new
```